firecloud_project.py

```diff
@@ -65,7 +65,8 @@ def iprange(number):
   "monitoring.googleapis.com",
   "storage-api.googleapis.com",
   "storage-component.googleapis.com",
-  "dns.googleapis.com"
+  "dns.googleapis.com",
+  "container.googleapis.com"
```

Comment:
`container.googleapis.com` is already near the top I think?

Also this looks like it's alphabetized :)

Reply:
ah it was from an old commit...